CONTRIBUTING.md

```diff
@@ -101,12 +101,19 @@ $ git checkout master
 $ git pull origin master
 $ git tag vX.X.X
 ```
-- Push it to the remote repository:
```

Comment:
What is the purpose of this push?

Reply:
if you don't push, the tag does not appear in the remote repository and will stay only locally.